kong/plugins/http-log/migrations/001_230_to_240.lua

```diff
@@ -0,0 +1,36 @@
+local operations = require "kong.db.migrations.operations.200_to_210"
```

Comment:
I reused this migrations file, is that ok, or should I create a new one with just the helper I need?

Reply:
@bungle is this ok, or do I need to duplicate it to a new version?

Reply:
We are supposed to create a new operations and copy-paste if we need to. The only reason to do that (as far as I know) is to minimize the risk from us modifying an existing function that was used on a previous migration by mistake (and not noticing until later ... when upgrading from an older version fails).

I guess if you are **very careful** and only use existing functions without adding new ones it's more or less ok... except for the fact that the module name is now completely **lying** - it says `200_to_210` but it is being used in a different migration. I find that ... irritating to be honest. Please copy the function to a different file just to make me happy.